PD-141 Add instructions on how to deal with NAS times that are wildly off (#1836)
truenas/documentation: content/SCALE/SCALETutorials/Dashboard/SCALETimeSync.md

- scaletimesync
---

TrueNAS SCALE allows users to synchronize SCALE and system server time when they get out of sync. 

> This function will not correct time differences over 30 days out of alignment.

Reviewer: active voice, present tense
```suggestion
This function does not correct time differences over 30 days out of alignment.
```

The **System Information** widget on the **[Dashboard]({{< relref "/SCALE/SCALEUIReference/SCALEDashboard.md" >}})** displays a message and provides an icon button that executes the time-synchronization operation only when SCALE detects a discrepancy between SCALE and system server time.

![DashboardSytemInfoWidgetTimeSync](/images/SCALE/22.12/DashboardSytemInfoWidgetTimeSync.png "System Information Widget with Time Sync")

Click the **Synchronize Time** <span class="material-icons">loop</span> icon button to initiate the time-synchronization operation.

{{< expand "What if my time is off by more than 30 days?" "v" >}}
If your time is off by more than 30 days, TrueNAS SCALE does not allow you to sync since the system probably has one of the following underlying issues:

* The BIOS timezone is incorrect
* The motherboard CMOS battery is failing

> To check the BIOS timezone, reboot your system. During boot, press the indicated key that takes you to the BIOS setup screen. The key differes based on manufacturer (<kbd>F2</kbd>, <kbd>Delete</kbd>, <kbd>Esc</kbd>, etc.). If you don't know which key to use, check the manufacturer documentation for your server.

Reviewer: no contractions for global audiences, one typo
```suggestion
To check the BIOS timezone, reboot your system. During boot, press the indicated key that takes you to the BIOS setup screen. The key differences based on manufacturer (<kbd>F2</kbd>, <kbd>Delete</kbd>, <kbd>Esc</kbd>, etc.). If you do not know which key to use, check the manufacturer documentation for your server.
```

> Once you have entered the BIOS setup, ensure the timezone is UTC. If not, set it to UTC, save the configuration changes, and reboot the system.

Reviewer: After instead of once is a better fit because they do this after entering the BIOS setup
```suggestion
After you enter the BIOS setup, ensure the timezone is UTC. If not, set it to UTC, save the configuration changes, and reboot the system.
```

A dying motherboard CMOS battery can also cause the system clock to be incorrect. If you intend to replace your CMOS, be sure to follow [electrostatic discharge (ESD) safety measures](https://www.wikihow.com/Ground-Yourself-to-Avoid-Destroying-a-Computer-with-Electrostatic-Discharge). 
{{< /expand >}}

> {{< expand "I corrected my system time issue. Why won't my apps start?" "v" >}}

Reviewer: while we don't want to use contractions, I'm leaving this because it is reasonable that someone would ask the question this way. If you can think of a better way to ask this question and keep it active voice, present tense and get rid of the contraction that is fine as well.

> If you corrected your system time issues (changed BIOS time, replaced CMOS, etc.) and your apps won't start, ensure all apps have their timezones set to *'UTC' timezone*.

Reviewer: ```suggestion
If you corrected your system time issues (changed BIOS time, replaced CMOS, etc.) and your apps do not start, ensure all apps have their timezones set to *'UTC' timezone*.
```
